compiler/passes/convert-typed-uast.cpp

```diff
@@ -4871,6 +4922,8 @@ bool TConverter::enter(const Call* node, RV& rv) {
     expr = x;
   } else if (auto x = convertNamedCallOrNull(node, rv)) {
     expr = x;
+  } else if (auto re = rv.byAst(node); re.type().isParam()) {
+    expr = new SymExpr(convertParam(re.type()));
```

Comment:
What is the difference between this branch and converting a param expression?

There is a branch somewhere at the top of e.g., `convertExpr` that tries to fold immediately if possible, maybe this branch should go there.